Changelog.md

```diff
@@ -0,0 +1,26 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
+
```

Comment:
Can you move this to Readme.md (as in diviapps)?

Then Changelog.md could be used to generate user facing changelog.